tests/query.test.js

```diff
@@ -2,23 +2,22 @@
 import expect from 'expect';
 import { create } from '../src/microstates';
 import { valueOf } from '../src/meta';
-import { map, filter, reduce } from '..';
+import { map, filter, reduce, find } from '..';
 
 import { TodoMVC } from './todomvc';
 
 describe('A Microstate with queries', function() {
   let todomvc;
-  beforeEach(function() {
-    let start = create(TodoMVC)
-      .todos.push({title: "Take out The Milk"})
-      .todos.push({title: "Convince People Microstates is awesome"})
-      .todos.push({title: "Take out the Trash"})
-      .todos.push({title: "profit $$"});
+  let start = create(TodoMVC)
```

Comment:
Only define the start variable in the describe block, the assignment should remain in beforeEach.

Reply:
oh okay